backend/fms_core/migrations/0070_v5_1_0.py

```diff
@@ -0,0 +1,138 @@
+
+from django.db import migrations
+from django.contrib.auth.models import User
+import reversion
+
+ADMIN_USERNAME = 'biobankadmin'
+
+UNKNOWN_INDEX_NAME = "UNKNOWN_INDEX"
+def create_empty_index(apps, schema_editor):
+    Index = apps.get_model("fms_core", "Index")
+    IndexStructure = apps.get_model("fms_core", "IndexStructure")
+    Sequence = apps.get_model("fms_core", "Sequence")
+
+    NO_FLANKERS_INDEX_STRUCTURE = {
+        "name": "No_Flankers_Index_Structure",
```

Comment:
Index_structure is the model you are creating. You can remove that part from the name.

Reply:
done.